src/tiledimage.js

```diff
@@ -873,17 +863,62 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
      * @returns {Boolean} Whether the TiledImage should be flipped before rendering.
      */
     getFlip: function() {
-        return !!this.flipped;
+        return this.flipped;
     },
 
     /**
      * @param {Boolean} flip Whether the TiledImage should be flipped before rendering.
      * @fires OpenSeadragon.TiledImage.event:bounds-change
      */
     setFlip: function(flip) {
-        this.flipped = !!flip;
+        this.flipped = flip;
+    },
+
+    get flipped(){
```

Comment:
Why the addition of getters and setters?

Reply:
`TiledImage.flipped` was already exposed as a public property, but it was presumably meant to be only read-only, because setting the value wouldn't have any effect - only `setFlip()` would actually cause relevant changes to propagate. By adding a getter/setter I tried to preserve the existing API while preemptively fixing what would have only been bugs before.

Reply:
Gotcha. Seems like a good strategy! Looks like these properties weren't documented, so they aren't technically public, but they also don't have the `_` prefix, so they aren't explicitly private. Since it's kind of a gray area, it's probably good to preserve them like this. 

As far as the API is concerned, I think we should stick with the pattern of having functions like `getFlip` and `setFlip` with `_` prefixed properties like `_flipped`. I think that's a clean, intuitive interface, even if it's not terribly modern. 

I see you've recently added a getter for the drawer type... Maybe that should be `getType()`?

Reply:
> I see you've recently added a getter for the drawer type... Maybe that should be `getType()`?

Sure, I can change those to `getType()`. I'll leave the getter/setter for `TiledImage.flip` as-is. Resolving this for now at least.